.github/workflows/main.yml

```
@@ -0,0 +1,25 @@
+name: OSX-build
+
+on: [push]
+
+jobs:
+  osx-clang:
+
+    name: OSX build
+    runs-on: [macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: setup
+      run: brew install libomp
+    - name: info
+      run: |
+        g++ -v
+        cmake --version
+    - name: configure
+      run: |
+        mkdir build
+        cd build
+        cmake ..
+        make -j 10
+        make -j 10 test
```

Does CMake automatically use 10 threads for the tests here? I think make test is single-threaded. How about something like 
```suggestion
        ctest -j 10
```